### DISCUSSION TITLE
Some issues regarding running Noah-MP with GLDAS forcing

### DISCUSSION THREAD

**Discussion by ZRYings** [NCAR/hrldas#103] [Ideas]
As I've been working through the GLDAS data processing, I encountered a few issues that I thought might be worth sharing. 

(1) The tutorial on GitHub (https://github.com/NCAR/hrldas/blob/master/hrldas/docs/README.GLDAS) suggests copying the create_forcing_netcdf.F file during the "Run the pre-processor" section (lines 168-176). However, I noticed that this file is not provided, making it impossible to proceed with this step.

(2) Additionally, in the provided create_forcing.F file (https://github.com/NCAR/hrldas/blob/master/hrldas/HRLDAS_forcing/create_forcing.F), line 445 reads: 'if ( ( (full_ic_frq>0) .and. (mod(ihour, full_ic_frq)==0) ) .or. ( ihour==0 ) ) then'. Since full_ic_frq is initially set to zero, this leads to overflowwhen trying to evaluate mod(ihour, full_ic_frq)==0.

(3) Another minor issue: when modifying the namelist.hrldas, it appears that the variable name for precipitation should be RAINRATE rather than PRCP. Since in the _vtable_, it refers to RAINRATE.

I'm not sure if these issues are widespread or specific to my case, but I would greatly appreciate it if they could be reviewed. In particular, regarding the issue at **line 445 in create_forcing.F**, would it be okay to directly remove the condition mod(ihour, full_ic_frq)==0?

Thanks.

Best regards,
Ruiying

**Comment by tslin2** [comment]
(1) create_forcing_netcdf.F was old code and it is not used anymore. Please use create_forcing.F
I will submit a PR soon to correct this file soon.
Thanks.

**Reply by tslin2** [reply]
(2) this issue looks had been addressed in this PR. https://github.com/NCAR/hrldas/pull/128
